aws/rust-runtime/aws-runtime/src/user_agent/interceptor.rs

```diff
@@ -99,6 +112,18 @@ impl Intercept for UserAgentInterceptor {
                 if let Some(app_name) = maybe_app_name {
                     ua.set_app_name(app_name.clone());
                 }
+
+                let maybe_connector_metadata = runtime_components
+                    .http_client()
+                    .and_then(|c| c.connector_metadata());
+                if let Some(connector_metadata) = maybe_connector_metadata {
+                    let fm = FrameworkMetadata::new(
```

Comment:
**fix**: This wouldn't be framework level metadata which is described as:

> For frameworks owned by the SDKs, e.g. `lib/chalice/1.2.3`

This is just additional `md/` pairs. e.g. `md/http#hyper-1.x`

Reply:
Got it. I'll update that.